xbmc/utils/DatabaseUtils.h

```diff
@@ -62,6 +62,10 @@ typedef enum {
   FieldAudioCount,
   FieldSubtitleCount,
 
+  FieldInstallDate,
```

Comment:
Nevermind. It's not about sorting, there are tests for the stupid cXX column names videoDatabase uses. So it's not relevant for us.